packages/docker-engine/Cargo.toml

```diff
@@ -12,6 +12,11 @@ path = "pkg.rs"
 url = "https://github.com/docker/engine/archive/v19.03.12/engine-19.03.12.tar.gz"
 sha512 = "51632ca8cff03440e3b76ad3e7f06b9c6275c776f4c4f9dc88cd7f5aa9b8aa8cb16ce2b3702426cb93f3714265a6cd28d4ba31612db37199e2915821b9ee8fd4"
 
+[[package.metadata.build-package.external-files]]
```

Comment:
A note here for why the patch is not checked in to the repo (like [this](https://github.com/bottlerocket-os/bottlerocket/blob/23a20e5e91cadc007a37f1c0471822c2fd2bf0d3/packages/kubernetes-1.17/Cargo.toml#L20-L22)) would be useful.

Reply:
I am not sure where should I check-in ? The patch should be merged to:  https://code.amazon.com/packages/DockerRpm/blobs/master/--/srpm/docker-19.03.6-ce-awslogs_update_aws-sdk-go_to_support_imdsv2.patch ?

Reply:
Sam just means adding a comment above this line explaining why the file isn't checked into the repo directly.  You can copy the comment he linked to start, and update it for these circumstances.  (Obviously the link in that comment doesn't apply directly, it's for a different package.)

Reply:
"This is a large patch, so we don't want to check it into the repo.  It's like https://github.com/docker/engine/commit/44a8e10bfc794dbbc4011a62fa8ba71d948d13cf and applies to 19.03.12 without changes."

Reply:
I confirmed it's exactly the same patch, so we don't need to say "like".  Perhaps:

```
# This is a large patch, so we don't want to check it into the repo. Source:
# https://github.com/docker/engine/commit/44a8e10bfc794dbbc4011a62fa8ba71d948d13cf
```